packages/mdx/readme.md

```diff
@@ -193,6 +193,10 @@ List of recma plugins.
 This is a new ecosystem, currently in beta, to transform [esast][] trees
 (JavaScript).
 
+###### `options.remarkRehypeOptions`
+
+Options for remark-rehype plugin.
```

Comment:
````suggestion
Options to pass through to [`remark-rehype`][remark-rehype].
The option `allowDangerousHtml` will always be set to `true` and the MDX nodes
are passed through.
In particular, you might want to pass `clobberPrefix`, `footnoteLabel`, and
`footnoteBackLabel`.

<details>
<summary>Example</summary>
```js
compile({
  value: '…',
  remarkRehypeOptions: {clobberPrefix: 'comment-1'}
})
```
</details>
````

(and at the bottom: `[remark-rehype]: https://github.com/remarkjs/remark-rehype`)